tests/scripts/create_subsets_aeronet.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Goal
+"""
+
+import os
+import shutil
+from functools import reduce
+from pathlib import Path
+
+import numpy as np
+
+import pyaerocom as pya
+
+OUTBASE = Path(pya.const._TESTDATADIR).joinpath("obsdata")
+
+if not OUTBASE.exists():
+    OUTBASE.mkdir()
+
+MIN_NUM_VALID = 300
+
+IDS = {
+    "sun": "AeronetSunV3Lev2.daily",
+    "sda": "AeronetSDAV3Lev2.daily",
+    "inv": "AeronetInvV3Lev2.daily",
+}
+
+NETWORKS = {"sun": "od550aer", "sda": "od550lt1aer", "inv": "abs550aer"}
+
+filters = [
+    ("altitude", [3000, 10000], 2),
+    ("altitude", [0, 1000], 10),
+    ("region_id", "OCN", 2),
+    ("station_name", "T*", 10),
+]
+
+revision_files = {}
+if __name__ == "__main__":
+
+    loaded = {}
+    for name, varlist in NETWORKS.items():
+        reader = pya.io.ReadUngridded()
+        _data = reader.read(IDS[name], varlist)
+        loaded[name] = _data
+        r = reader.get_reader()
+        revision_files[name] = Path(r.DATASET_PATH).joinpath(r.REVISION_FILE)
+
+    use_stats = []
+
+    for (attr, val, maxnum) in filters:
+        subsets = {}
+        statnames = []
+
+        for name, data in loaded.items():
+
+            subset = data.apply_filters(**{attr: val})
+
+            subsets[name] = subset
+            statnames.append(subset.unique_station_names)
+
+        # same_stats = np.intersect1d(*statnames)
```

Comment:
please remove dead code

Reply:
I put that into a separate issue #604. A bit out of scope for now.